javagen/src/main/java/com/azure/autorest/template/example/ModelExampleWriter.java

```diff
@@ -146,6 +146,7 @@ public void accept(ExampleNode node, String getterCode) {
                 }
             } else if (node instanceof BinaryDataNode) {
                 this.imports.add(com.azure.core.util.BinaryData.class.getName());
+                this.imports.add(java.nio.charset.StandardCharsets.class.getName());
                 addEqualsAssertion(binaryDataNodeExpression((BinaryDataNode) node), getterCode);
```

Comment:
Are you sure this is correct?

There is no `equals` in `BinaryData`?

Just assertNotNull?

Reply:
I see... Just notNull then...